backend/app/Console/Commands/MqttListenCommand.php

```
@@ -10,143 +10,221 @@
+        $client = $this->buildClient($cfg);
+        $client->connect($this->buildSettings($cfg), $cfg->clean_session);
+
+        foreach ($conn->activeTopics as $topic) {
+            $client->subscribe(
+                $topic->topic_pattern,
+                function (string $incomingTopic, string $message) use ($conn, $dryRun) {
+                    $receivedAt = now()->toIso8601String();
+                    $this->line("[{$conn->id}] {$incomingTopic}: ".substr($message, 0, 120));
+
+                    if (! $dryRun) {
+                        ProcessMqttMessageJob::dispatch($conn->id, $incomingTopic, $message, $receivedAt);
```

PHPStan (2.2.5)

[error] Call to an undefined static method App\Jobs\ProcessMqttMessageJob::dispatch().


(staticMethod.notFound)